docs/source/troubleshooting.rst

```diff
@@ -18,10 +18,22 @@ Building
 Flashing
 --------
 
-* The board is not flashing.
+*Here are some of the tips if The board is not flashing..*
+
+* Sometimes to program ESP32 via serial you must keep GPIO0 LOW during the programming process.
```

Comment:
In some instances, you must keep GPIO0 LOW during the uploading process via serial interface.

Reply:
Ok, Done.